searchcore/src/vespa/searchcore/proton/server/documentdb.cpp

```diff
@@ -175,7 +175,7 @@ DocumentDB::DocumentDB(const vespalib::string &baseDir,
       _baseDir(baseDir + "/" + _docTypeName.toString()),
       // Only one thread per executor, or performDropFeedView() will fail.
       _writeServiceConfig(configSnapshot->get_threading_service_config()),
-      _writeService(shared_service.shared(), _writeServiceConfig, indexing_thread_stack_size),
+      _writeService(shared_service.shared(), shared_service.field_writer(), _writeServiceConfig, indexing_thread_stack_size),
```

Comment:
I assume just passing shared_service is part of cleanup once completed.

Reply:
Yes. We might also consider keeping it like this as the SharedThreadingService contains more than the ExecutorThreadingService needs.